frontend/src/app/modules/ticket/pages/ticket-add/ticket-add.component.ts

```diff
@@ -0,0 +1,42 @@
+import { Component } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { TicketService } from '../../services/ticket.service';
+import { Ticket } from '../../models/Ticket';
+import { FormBuilder } from '@angular/forms';
+import { Validators } from '@angular/forms';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-ticket-add',
+  templateUrl: './ticket-add.component.html',
+  styleUrls: ['./ticket-add.component.scss']
+})
+export class TicketAddComponent {
+
+  constructor(
+    private ticketService : TicketService,
+    private fb : FormBuilder,
+    private router : Router){}
+  
+    ticketToCreate = new Ticket;
+
+   ticketAdditionForm = this.fb.group({
```

Comment:
prenez l'habitude de bien faire passer le formatter sur votre code pour garder une structure homogène du code dans tout le projet https://stackoverflow.com/questions/29973357/how-do-you-format-code-in-visual-studio-code-vscode